### DISCUSSION TITLE
Fedimint block explorer

### DISCUSSION THREAD

**Discussion by justinmoon** [fedimint/fedimint#4076] [General]
One thing that might really help developers explore fedimint is a fedimint block explorer. It could either directly read the federation log from fedimintd guardian database, or join the federation as a client and read the federation log via RPC. Being able to create different types of transactions on clients and visualize what the transactions actually contain would really help people better understand.

For the default modules I think this would be pretty simple. Supporting arbitrary external modules will once again be the hard part.

**Comment by Kodylow** [comment]
This would be a super interesting UI project for someone

**Comment by bradleystachurski** [comment]
Related: https://github.com/fedimint/fedimint/issues/3843
